mopidy_local/storage.py

```diff
@@ -58,6 +58,14 @@ def get_image_size_jpeg(data):
     return width, height
 
 
+def test_jpeg(h, f):
```

Comment:
I had to check the docs for `imghdr`. Maybe rename to something like this?

```suggestion
def test_jpeg(data, file_handle):
```